migrations/036-add_customers_table.js

```diff
@@ -0,0 +1,32 @@
+var db = require('./db')
+
+exports.up = function(next) {
+  const sql =
+    [
+    'CREATE TABLE customers ( ' +
+    'id uuid PRIMARY KEY,' +
+    'phone text unique NOT NULL,' +
+    'phone_at timestamptz NOT NULL,' +
```

Comment:
I realized that since cash_in_txs has a foreign key to this table, we always need to link to a record in this table. Since it's possible to have a cash_in_tx where we don't know the user's phone number, we should make these fields optional.